### PR TITLE
move flyscanning document from ophyd-async

### PR DESCRIPTION
**Pull request by stan-dot**

### CODE REVIEW

File: glossary/flyscanning.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Flyscanning (also known as hardware triggered scanning, asynchronous acquisition, and hardware synchronized scanning) is the practice of accelerating the rate of data collection by handing control over to an external hardware system that can control and synchronize the triggering of detectors with other signals and/or commands. Flyscans take many forms.
+
+.. _detectorsync_:
+
+Detector Synchronization
+------------------------
+
+.. figure:: ../images/simple-hardware-scan.png
+    :alt: hardware-triggered setup
+    :width: 300
```
Comment (callumforrester) [inline glossary/flyscanning.md:13]: The source also needs to be moved to this repo

File: glossary/flyscanning.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+A triggering system can send pulses to two or more detectors to make them expose simultaneously, or at different multiples of the same base rate (e.g. 200Hz and 400Hz).
+
+.. _motortraj_:
+
+Motor Trajectory Scanning
+-------------------------
+ 
+.. figure:: ../images/hardware-triggered-scan.png
+    :alt: trajectory scanning setup
```
Comment (callumforrester) [inline glossary/flyscanning.md:23]: The source also needs to be moved to this repo

File: glossary/flyscanning.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+.. _outerscan_:
+
+Outer Scanning
+--------------
+
+Outer scans are flyscans nested inside soft scans. 
+
+.. figure:: ../images/outer-scan.png
+    :alt: hardware-triggered setup
```
Comment (callumforrester) [inline glossary/flyscanning.md:35]: The source also needs to be moved to this repo

File: glossary/flyscanning.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Ophyd-async's job is to provide devices that implement ``Flyer`` and can:
+
+- Configure all necessary hardware for a scan
+- Kickoff a scan and monitor progress until complete
+- Produce documents representing the progress of the scan
+- Allow handing control back and forth to enable outer scanning
+
+.. _PandA: https://quantumdetectors.com/products/pandabox/
+.. _Zebra: https://quantumdetectors.com/products/zebra/
```
Comment (callumforrester) [inline glossary/flyscanning.md:63]: I think these two sections still belong in the ophyd-async docs, they are not definitions but explanations of how ophyd-async works and what it supports.

**Review by callumforrester — CHANGES_REQUESTED** [review]
Please also raise a corresponding PR in ophyd-async removing what you add here so we can merge both at the same time

**Comment by callumforrester** [comment]
@stan-dot these files are rst syntax with an md extension, which were you going for?

File: glossary/flyscanning.md (hunk 4 above)
Comment (stan-dot) [reply glossary/flyscanning.md:63]: ok, will make an exception

**Comment by stan-dot** [comment]
>  rst syntax with an md extension, 
@callumforrester I think I started converting but not all got changed. I'm on it

File: glossary/flyscanning.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Flyscanning
```
Comment (callumforrester) [inline glossary/flyscanning.md:1]: Hmm, this is a definition like all the others but is also the page title. I'd like the flyscanning definition to be consistent with the others but the following seems pretty redundant:

```suggestion
# Flyscanning

## Flyscanning
```

Any thoughts?

**Review by callumforrester — CHANGES_REQUESTED** [review]

File: glossary/glossary-todo.md
[the remark this answers is not in the document]
Comment (stan-dot) [reply glossary/glossary-todo.md:1]: Sure thing

File: glossary/flyscanning.md (hunk 5 above)
Comment (stan-dot) [reply glossary/flyscanning.md:1]: Ok I'll, change that

**Review by callumforrester — CHANGES_REQUESTED** [review]
Need to rebase on top of main and check CI passes

**Comment by stan-dot** [comment]
depends on pr #17

**Pull request merged**